ibmjava/8/jre/ubi-std/Dockerfile

```diff
@@ -0,0 +1,81 @@
+# (C) Copyright IBM Corporation 2016, 2018
```

Comment:
I'd prefer if we use the same naming convention as RedHat for this viz `ubi` instead of `ubi-std`.